### PR TITLE
Fixed wide image shown as regular if screen width below 1040px

### PR DESCRIPTION
**Pull request by marcoSven**
Fixed null values for CSS properties

### CODE REVIEW

File: packages/gatsby-rehype-inline-images/src/gatsby-theme-try-ghost/styles/fluid.css [hunk 1]
```diff
@@ -3,16 +3,14 @@
 /* Fluid inline images
 /* ---------------------------------------------------------- */
 
-html.casper .post-content figure {
+html.casper .post-content {
+    display: flex!important;
```
Comment (marcoSven) [inline packages/gatsby-rehype-inline-images/src/gatsby-theme-try-ghost/styles/fluid.css:7]: I think we can remove !important if we remove this adjustment https://github.com/styxlab/gatsby-theme-try-ghost/blob/master/packages/gatsby-theme-ghost-toc/src/gatsby-theme-try-ghost/styles/toc.css#L23

File: packages/gatsby-rehype-inline-images/src/gatsby-theme-try-ghost/styles/fluid.css [hunk 2]
```diff
@@ -24,3 +22,28 @@ html.casper .kg-bookmark-thumbnail .gatsby-image-wrapper {
     border-radius: 0 3px 3px 0;
     object-fit: cover;
 }
+
+@media (max-width:1040px) {
+    html.casper .post-content {
+        display: block!important;
```
Comment (marcoSven) [inline packages/gatsby-rehype-inline-images/src/gatsby-theme-try-ghost/styles/fluid.css:28]: same here

**Comment by styxlab** [comment]
That's a brilliant change - I will do some more testing before merging, but it looks good so far.

**Review by styxlab — APPROVED** [review]
As I do not want to change the originals, using the css !important modifier is the only escape hatch. This PR is about wide images and they work as expected, so this change is approved.

During testing, I noticed that full images are not taking the full screen width, e.g.:
image

Note that the width of the 3rd picture is the same as the second.

**Comment by styxlab** [comment]
Fixes #230

**Pull request merged**